app/src/themes/appTheme.ts

```diff
@@ -43,6 +47,14 @@ const appTheme = createMuiTheme({
         fontSize: '1rem'
       }
     },
+    // typography: {
+    //   body: {
+    //     fontFamily: "'Open Sans', sans-serif",
+    //     fontWeight: 400,
+    //     fontSize: 16,
+    //     color: "red"
+    //   }
+    // },
```

Comment:
Looks like this is captured above, may as well remove this.

Reply:
done

Reply:
Looks like this got missed